<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>941</width>
      <height>741</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../main8.png</pixmap>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>741</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditID1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSoban">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>580</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditID2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>550</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonXN1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>660</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>XÁC NHẬN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonXN2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>650</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>XÁC NHẬN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TRANG CHỦ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ĐĂNG XUẤT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHuy">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>660</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HỦY</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
